src/main/java/eggy/ui/Ui.java

```diff
@@ -109,6 +110,18 @@ public void printTaskListEmpty() {
         System.out.println("     There are no tasks in your list.");
     }
 
+    /**
+     * Prints the matching tasks.
+     *
+     * @param tasks The matching tasks.
+     */
+    public void printMatchingTasks(List<Task> tasks) {
+        System.out.println("     Here are the matching tasks in your list:");
+        for (int i = 0; i < tasks.size(); i++) {
+            System.out.println("     " + (i + 1) + "." + tasks.get(i));
+        }
+    }
```

Comment:
May I ask why there's a space before printing the corresponding task number and description?

Reply:
I followed the exact indentation from the project guidelines example.